CHANGES.rst

```diff
@@ -57,6 +57,9 @@ Mosviz
 Specviz
 ^^^^^^^
 
+- Only combine multi-order x1d spectra into a concatenated spectrum for MIRI (and not
+  for NIRISS, for example) [#1659]
```

Comment:
this sounds like its automated, but this is currently entirely controlled by the user via `concat_by_file`, right?

Reply:
I agree, editing the log now.

Reply:
Did the edit happen? I don't see any new commit.